cocos/2d/CCLabel.cpp

```diff
@@ -47,6 +47,13 @@
 
 NS_CC_BEGIN
 
+#if CC_LABEL_DEBUG_DRAW
```

Comment:
https://docs.microsoft.com/en-us/cpp/preprocessor/hash-ifdef-and-hash-ifndef-directives-c-cpp?view=vs-2017

Reply:
I added code to leave the behaviour of the debug helpers as it was originally, but still keeping the ability to enable or disable them if set a preprocessor definition to use them

No new macros have been added

And as any can review, the few #if CC_LABEL_DEBUG_DRAW  are following the way the code was using it before this branch